recipe/meta.yaml

```diff
@@ -22,17 +22,17 @@ requirements:
     - ipython >=5.0
     - jupyter_client
```

Comment:
Do we need to force a pin here from the recent client releases w/r/t py38/win issues?

Reply:
I'd just make this match setup.py